### ISSUE TITLE
Insufficient color contrast in page header cart count badge

### ISSUE THREAD

**Issue by hackartisan** [pulibrary/pulfalight#1737]
refs #1717 
[full list of issues for audit1717](https://github.com/pulibrary/pulfalight/issues?q=is%3Aissue%20state%3Aopen%20audit1717)

### Location of problem on site

Page heading, request cart icon count badge

### Description of error / warning

The white number on the red background has an insufficient contrast, measuring 3.99:1. It should be 4.5:1

### WCAG success criterion

[1.4.3 Contrast (Minimum)](https://webaim.org/standards/wcag/checklist#sc1.4.3)

### WCAG conformance level

Level AA

### Recommended fix

Darken or change the red background

### Tool used to find the problem

wave

### Screenshot

<img width="657" height="160" alt="Image" src="https://github.com/user-attachments/assets/0380f2d6-3fb8-4288-9c4b-3ebf5248b90e" />
